accounts/verification.go

```diff
@@ -25,77 +26,88 @@ const (
 func generateBase64Token() string {
 	tokenBytes := make([]byte, tokenLength)
 	rand.Read(tokenBytes)
-	token := b64.StdEncoding.EncodeToString(tokenBytes)
-	return token
+	// remove slashes as verification code will be read as url param
+	removeCharByte(tokenBytes, byte(63))
+	return b64.StdEncoding.EncodeToString(tokenBytes)
+}
+
+func randomInt(min int, max int) int {
```

Comment:
This seems like a utility function that may/can be used elsewhere? I don't think it's a problem if we leave it here, but if you know a use case where we'd use it again in the future, maybe we can move this out of verification. Same with `removeCharByte` below.